src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMScoring.h

```diff
@@ -233,6 +237,9 @@ namespace OpenSwath
         const OpenMS::Matrix<double> & getMIPrecursorCombinedMatrix() const;
         //@}
 
+        void fillIntensityFromFeature(OpenSwath::IMRMFeature* mrmfeature, const std::vector<String>& ids, std::vector<std::vector<double>>& intensity);
```

Comment:
can you document these methods? including parameter

Reply:
Thanks for adding documentation. Some general hints to improve it.

1. Using doxygen notation, you can highlight parameters. Without @param it is hard to understand what is a textual description and what are actual variable names. 
2. Mentioning "feature" might be a bit redundant and the sentence could be rephrased. 
3. But most importantly, the comment doesn't tell anything in addition what is already written. A better comment would tell a user what to pass as input (e.g., what do I pass a pointer or is this a C style array?)  What are the ids? Are these peak group ids? transition ids? And what is the output? Is this a matrix? What does it represent (e.g., chromatograms? of the same MRMFeature? or of several ones listes as ID.

You see that the current comment leaves a lot of open questions that all might be answered in one comment line.